nbconvert/exporters/exporter.py

```diff
@@ -84,7 +84,6 @@ class Exporter(LoggingConfigurable):
                                   'nbconvert.preprocessors.ExecutePreprocessor',
                                   'nbconvert.preprocessors.coalesce_streams',
                                   'nbconvert.preprocessors.SVG2PDFPreprocessor',
-                                  'nbconvert.preprocessors.CSSHTMLHeaderPreprocessor',
```

Comment:
Ultimately, we probably should remove all exporter-specific preprocessors from the defaults, and move them up to
 - `@default(preprocessor)` in exporters
 - or further up to the template configuration for template exporters.